api/controller/trigger_metrics_test.go

```diff
@@ -328,3 +328,218 @@ func TestGetTriggerMetrics(t *testing.T) {
 		So(triggerMetrics, ShouldBeNil)
 	})
 }
+
+func TestSetTriggerCheckMaintenance(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+	dataBase := mock_moira_alert.NewMockDatabase(mockCtrl)
+
+	triggerID := uuid.Must(uuid.NewV4()).String()
+	trigger := moira.Trigger{ID: triggerID}
+
+	emptyLastCheck := moira.CheckData{
+		Metrics: make(map[string]moira.MetricState),
+	}
+
+	Convey("While no check", t, func() {
```

Comment:
What is the reason to avoid top-level `Convey` call? In file `database/redis/last_check_test.go` all this test cases was in one huge `Convey`